app/helpers/users_helper.rb

```diff
@@ -0,0 +1,12 @@
+module UsersHelper
+  def gravatar_url(user)
+    hash = Digest::MD5.hexdigest(user.email)
+    "https://www.gravatar.com/avatar/#{hash}"
+  end
+
+  def has_gravatar?(user)
+    gravatar_url = gravatar_url(user)
+    response = HTTParty.get(gravatar_url, query: {d: '404'}).code
+    return response != 404
+  end
```

Comment:
Hmm, on second thought let's just fall back on Gravatar's default image if they don't have a Gravatar set. Making an HTTP request on the server every page load seems like too much (and the page load will fail if the request throws an exception).